tests/test_all.py

```diff
@@ -88,6 +88,20 @@ def test_os_release(auto_container):
         )
 
 
+def test_branding(auto_container):
+    """
+    check that the :file:`/etc/SUSE-brand` file exists and containers SLE branding
+    """
+    assert auto_container.connection.file("/etc/SUSE-brand").exists
```

Comment:
The location is `/usr/etc/SUSE-brand` on Tumbleweed.

Reply:
gah